R/predictDirectedNetworks.R

```diff
@@ -0,0 +1,63 @@
+# predictDirectedNetworks.R
+
+source('./R/fetchData.R', echo=TRUE)
```

Comment:
No need to source those files, as they are installed and loaded whenever as part of the package

Reply:
Fixed.